test/norm_num.lean

```diff
@@ -2,12 +2,14 @@
 Copyright (c) 2017 Simon Hudon All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Simon Hudon, Mario Carneiro
-
-Tests for norm_num
 -/
 
 import tactic.norm_num
 
+/--
```

Comment:
```suggestion
/-!
```

We're not rendering these with doc-gen at the moment, but it doesn't hurt to be consistent.

Reply:
Oh yeah of course. Must've had jam in my head.